WcaOnRails/app/models/competition.rb

```diff
@@ -1161,6 +1159,14 @@ def confirmed=(new_confirmed_str)
     self.confirmed_at = new_confirmed ? (self.confirmed_at || Time.now) : nil
   end
 
+  def is_using_wca_live_for_scoretaking
```

Comment:
Why are you defining a method that is just redirecting to a field, instead of accessing that field directly?
Also, Ruby has a strong convention that all methods returning booleans should end in a question mark.